bazaarvoice/maven-cassandra-plugin
# PD-179033 support C* 3.11.x

## Github Issue #

-

## What Are We Doing Here?

support C* 3.11.x

## How to Test and Verify

1. Check out this PR
2. `mvn -e clean verify  -Ddependency-check.skip=true -Prun-its`
3. Profit

## Risk

### Level 

`Low`

### Required Testing

`Manual`

### Risk Summary

no issues are expected. the plugin is used mostly for integration tests.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
